### ISSUE TITLE
document uiautomator events logging command

### ISSUE THREAD

**Issue by bootstraponline** [appium/ruby_lib#319]
`adb shell uiautomator events` is useful when debugging to see what accessibility events are occuring on the device in response to appium commands. This should be documented somewhere.

**Comment by bootstraponline** [comment]
If appium is started then it terminates the other uiautomator processes :cry:
